hometask7/src/game.ts

```diff
@@ -0,0 +1,80 @@
+import { Categories } from "./enums";
+import { checkEven } from "./helpers";
+import { Questionnaire } from "./questionnaire";
+import { Player } from "./player";
+import { Players } from "./players";
+import { logger } from "./logger";
+
+export class Game {
+  private questionnaire: Questionnaire;
+  private players: Players;
+  private player: Player;
+  private winCount: number = 5;
+
+  constructor(players: Player[]) {
+    this.questionnaire = new Questionnaire(Categories, 50);
+    this.players = new Players(players);
+    this.player = this.players.current;
+  }
+
+  public addPlayer(player: Player) {
+    this.players.add(player);
+  }
+
+  public run() {
+    this.roll();
+  }
+
+  private roll() {
+    const roll = this.generateRoll();
+    const isEven = checkEven(roll);
+
+    if (this.player.inPenaltyBox && isEven) {
+      this.player.staysInPenalty();
+    }
+
+    if (this.player.inPenaltyBox && !isEven) {
+      this.player.goesOutOfPenalty();
+    }
+
+    if (this.player.inPenaltyBox) {
```

Comment:
TODO: 3 if conditions with the same: `this.player.inPenaltyBox`. Please refactor